templates/index.css

```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  padding: 0;
+}/*# sourceMappingURL=index.css.map */
```

Comment:
What are these files?

Reply:
I think these are what 'Watch Sass' is generating after I save a .scss file